tests/ci/run_check.py

```diff
@@ -127,13 +127,10 @@ def should_run_checks_for_pr(pr_info):
 def check_pr_description(pr_info):
     description = pr_info.body
 
-    lines = [
-        line
-        for line in map(
-            lambda x: x.strip(), description.split("\n") if description else []
-        )
-    ]
-    lines = [re.sub(r"\s+", " ", l) for l in lines]
+    lines = list(
+        map(lambda x: x.strip(), description.split("\n") if description else [])
+    )
+    lines = [re.sub(r"\s+", " ", line) for line in lines]
```

Comment:
I'd prefer just to use comprehensions instead of `map` Also we can do `re.sub` in loop below.
Also haven't checked suggested lines, do we have test for this code?
But actually I suppose it doesn't matter here, let's leave your version.
```suggestion
    lines = description.split("\n") if description else []
    lines = [re.sub(r"\s+", " ", line.strip()) for line in lines]
```